release-notes/System.ServiceModel.Primitives.md

```diff
@@ -0,0 +1,15 @@
+# System.ServiceModel.Primitives
+
+## Information
+
+* [NuGet Package](https://nuget.org/packages/System.ServiceModel.Primitives)
+* [Report an issue](http://github.com/dotnet/wcf/issues/new)
+* [Source](https://github.com/dotnet/wcf/tree/master/src)
+
+## Version History
+
+### 4.1.0-beta-23516 
+
+* [Release tag](https://github.com/dotnet/wcf/tree/v1.0.0-rc1)
+* [Supported features](https://github.com/dotnet/wcf/blob/master/release-notes/SupportedFeatures-v1.0.0-rc1.md)
+* Adds new public API surface area to enable use of X509Certificate2
```

Comment:
Excellent question, especially because "what's new" doesn't always correlate well with one of these 5 packages.  We don't have a System.Private.ServiceModel.md (should we?) because I thought it would be confusing.  But that's logically where most of the "what's new" belongs. I merged this PR to keep moving forward, but I think the correct approach should be to update each individual package .md file with anything new *in that package* but to describe new features and major bug fixes in the single https://github.com/dotnet/wcf/releases page.  Thoughts?